pwndbg/commands/context.py

```diff
@@ -670,12 +737,19 @@ def context(subcontext=None, enabled=None) -> None:
             with target as out:
                 res.append(pwndbg.ui.banner("", target=out, width=settings.get("width", None)))
 
+    cmd_lines = 0
```

Comment:
More complexity to the `context` output 😅 ! ;D

Heh, this func is a bit bloated, I started refactoring it recently but did not finish it ;<